mkdocs/docs/experiments/categories/pods/pod-dns-error.md

```diff
@@ -4,29 +4,26 @@
 - It causes loss of access to services by blocking dns resolution of hostnames/domains
 
 !!! tip "Scenario: DNS error for the target pod"
-    ![Pod DNS Error](../../images/dns-chaos.png)
+![Pod DNS Error](../../images/dns-chaos.png)
 
 ## Uses
 
 ??? info "View the uses of the experiment"
-    coming soon
+coming soon
 
 ## Prerequisites
 
-??? info "Verify the prerequisites"
-    - Ensure that Kubernetes Version > 1.16
-    -  Ensure that the Litmus Chaos Operator is running by executing <code>kubectl get pods</code> in operator namespace (typically, <code>litmus</code>).If not, install from <a href="https://v1-docs.litmuschaos.io/docs/getstarted/#install-litmus">here</a>
-    -  Ensure that the <code>pod-dns-error</code> experiment resource is available in the cluster by executing <code>kubectl get chaosexperiments</code> in the desired namespace. If not, install from <a href="https://hub.litmuschaos.io/api/chaos/master?file=charts/generic/pod-dns-error/experiment.yaml">here</a>
+??? info "Verify the prerequisites" - Ensure that Kubernetes Version > 1.16 - Ensure that the Litmus Chaos Operator is running by executing <code>kubectl get pods</code> in operator namespace (typically, <code>litmus</code>).If not, install from <a href="https://v1-docs.litmuschaos.io/docs/getstarted/#install-litmus">here</a> - Ensure that the <code>pod-dns-error</code> experiment resource is available in the cluster by executing <code>kubectl get chaosexperiments</code> in the desired namespace. If not, install from <a href="https://hub.litmuschaos.io/api/chaos/master?file=charts/generic/pod-dns-error/experiment.yaml">here</a>
```

Comment:
why we removed newline here?

Reply:
My vscode formatted it, reverting back the changes :)